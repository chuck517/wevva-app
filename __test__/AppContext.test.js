import React from 'react';
import renderer from 'react-test-renderer';
import AppProvider, { addCity, savedCityList, setSavedCityList } from '../AppContext';

it('renders without crashing', () => {
  const rendered = renderer.create(<AppProvider />).toJSON();
  expect(rendered).toBeTruthy();
});

it('test against snapshot', () => {
  const tree = renderer.create(<AppProvider />).toJSON();
  expect(tree).toMatchSnapshot();
});