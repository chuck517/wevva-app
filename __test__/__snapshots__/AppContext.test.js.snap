// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test against snapshot 1`] = `null`;
